src/api.rs

```diff
@@ -125,12 +96,12 @@ mod test {
     #[test]
     fn reason() {
         let mut engine =
-            load("./resources/testcases/lcs-diff-computation/run-lcs-10.rls".to_string()).unwrap();
+            load("./resources/testcases/lcs-diff-computation/run-lcs-10.rls".into()).unwrap();
         let cur_dir = std::env::current_dir().unwrap();
         std::env::set_current_dir("./resources/testcases/lcs-diff-computation/").unwrap();
         let results = super::reason(&mut engine).unwrap();
         std::env::set_current_dir(cur_dir).unwrap();
-        assert_eq!(results.len(), 20); // 2 empty tables, which are omitted
+        assert_eq!(results.len(), 22); // 2 empty tables, which are _not_ omitted
```

Comment:
That comment is not very informative now.
The previous version removed all IDBPredicates without table_ids from the result, so it was reasonable to mention that 2 empty tables are omitted.
As this filtering is no longer in place, it is not needed to mention that the two empty tables are not omitted.